src/reanimated2/platform-specific/checkVersion.ts

```diff
@@ -0,0 +1,28 @@
+import { version as jsVersion } from '../../../package.json';
```

Comment:
webpack won't like this since JSON is treated as a default import. it should be `import pkg` and then do `pkg.version` later on.

Reply:
Disregard, just saw this is a native-only file.